### PR TITLE
Corrections to example responses for issue #334

### PR DESCRIPTION
**Pull request by m-burgoyne**
Updated the example responses to fix problems with the links sections and removed obsolete documentation

### CODE REVIEW

File: standard/openapi/examples/instance.json [hunk 1]
```diff
@@ -805,24 +787,15 @@
...
                 {
-                    "href": "http://www.example.org/edr/collections/3_hrly_fcst/instances/2020-06-30T10:00:00Z/position",
-                    "hreflang": "en",
-                    "rel": "data"
-                },
-                {
-                    "href": "http://www.example.org/edr/collections/3_hrly_fcst/instances/2020-06-30T10:00:00Z/radius",
+                    "href": "http://www.example.org/edr/collections/3_hrly_fcst/instances/2020-06-30T08:00:00Zf=json",
                     "hreflang": "en",
-                    "rel": "data"
```
Comment (tomkralidis) [inline standard/openapi/examples/instance.json:815]: no relation type for this link?

File: standard/openapi/examples/instance.json [hunk 2]
```diff
@@ -805,24 +787,15 @@
...
+                    "type": "application/json"
                 },
                 {
-                    "href": "http://www.example.org/edr/collections/3_hrly_fcst/instances/2020-06-30T10:00:00Z/area",
+                    "href": "http://www.example.org/edr/collections/3_hrly_fcst/instances/2020-06-30T08:00:00Zf=html",
                     "hreflang": "en",
-                    "rel": "data"
```
Comment (tomkralidis) [inline standard/openapi/examples/instance.json:820]: Same here, no relation type for this link?

File: standard/openapi/examples/instance.json [hunk 3]
```diff
@@ -1178,24 +1151,15 @@
...
-                {
-                    "href": "http://www.example.org/edr/collections/3_hrly_fcst/instances/2020-06-30T10:00:00Z/radius",
-                    "hreflang": "en",
-                    "rel": "data"
-                },
-                {
-                    "href": "http://www.example.org/edr/collections/3_hrly_fcst/instances/2020-06-30T10:00:00Z/area",
+                    "href": "http://www.example.org/edr/collections/3_hrly_fcst/instances/2020-06-30T07:00:00Zf=json",
                     "hreflang": "en",
-                    "rel": "data"
```
Comment (tomkralidis) [inline standard/openapi/examples/instance.json:1193]: And here, no relation type for this link?

File: standard/openapi/examples/instance.json [hunk 4]
```diff
@@ -59,24 +59,15 @@
             ],
             "links": [
                 {
-                    "href": "http://www.example.org/edr/collections/3_hrly_fcst/instances/2020-06-30T10:00:00Z/position",
+                    "href": "http://www.example.org/edr/collections/3_hrly_fcst/instances/2020-06-30T10:00:00Z?f=json",
                     "hreflang": "en",
-                    "rel": "data"
```
Comment (tomkralidis) [inline standard/openapi/examples/instance.json:64]: Here too, no relation type for this link?

File: standard/openapi/examples/collection.json [hunk 5]
```diff
@@ -50,31 +50,28 @@
...
+                    "href": "http://www.example.org/data/datapoint/terms-and-conditions---datapoint#termsofservice",
                     "hreflang": "en",
                     "rel": "restrictions",
                     "type": "text/html"
                 },
                 {
-                    "href": "http://www.example.org/edr/collections/hrly_obs/position",
+                    "href": "http://www.example.org/edr/collections/hrly_obs?f=json",
                     "hreflang": "en",
-                    "rel": "data"
```
Comment (tomkralidis) [inline standard/openapi/examples/collection.json:61]: Same here, no relation type for this link?

**Review by tomkralidis — CHANGES_REQUESTED** [review]
Thanks @m-burgoyne.  Extremely minor nitpick (maybe for another PR): normalize all "example" links to use https://example.org as the base.

File: standard/openapi/examples/collection.json (hunk 5 above)
Comment (m-burgoyne) [reply standard/openapi/examples/collection.json:61]: @tomkralidis, they should be collection responses I have updated the pull request

File: standard/examples/instance_metadata_JSON.adoc [hunk 6]
```diff
@@ -821,24 +818,22 @@ There are also links to the license information for the observation and forecast
...
-                    "href": "http://http://www.example.org/edr/collections/3_hrly_fcst/instances/2020-06-30T10:00:00Z/radius",
+                    "href": "http://http://www.example.org/edr/collections/3_hrly_fcst/instances/2020-06-30T08:00:00Z",
                     "hreflang": "en",
-                    "rel": "data"
+                    "rel": "self",
+                    "type": "application/json"
                 },
                 {
-                    "href": "http://http://www.example.org/edr/collections/3_hrly_fcst/instances/2020-06-30T10:00:00Z/area",
+                    "href": "http://http://www.example.org/edr/collections/3_hrly_fcst/instances/2020-06-30T08:00:00Z?f=html",
```
Comment (tomkralidis) [inline standard/examples/instance_metadata_JSON.adoc:827]: Just caught many instances of this (double protocol).  Maybe normalizing to https://example.org is better done in this PR?  Your call.

Comment (m-burgoyne) [reply standard/examples/instance_metadata_JSON.adoc]: @tomkralidis Thanks I had missed that

**Review by tomkralidis — APPROVED** [review]

**Pull request merged**